cms/envs/production.py

```diff
@@ -617,6 +617,19 @@
 # We continue to load production.py over aws.py
 plugin_settings.add_plugins(__name__, plugin_constants.ProjectType.CMS, plugin_constants.SettingsType.PRODUCTION)
 
+########################## Philu related settings ########################
```

Comment:
We've all these setting in `aws.py` would you please elaborate why those settings are not working now. Also if those are no more required please remove them from `aws.py`

Reply:
if you open `aws.py` you can see this warning `logging.warn('DEPRECATION WARNING: aws.py has been deprecated, you should use production.py instead.')` That's why I put these settings in `production.py`.